a10_octavia/etc/example_config.py

```diff
@@ -0,0 +1,68 @@
+[DEFAULT]
```

Comment:
If we are going to be treating these config files as text files, then we need to name them `.conf` instead of `.py`